includes/class-sensei-course.php

```diff
@@ -2459,12 +2459,16 @@ public static function get_course_loop_content_class() {
 		$sensei_course_loop['counter']++;
 
 		$extra_classes = array();
-		if ( 0 == ( $sensei_course_loop['counter'] - 1 ) % $sensei_course_loop['columns'] || 1 == $sensei_course_loop['columns'] ) {
-			$extra_classes[] = 'first';
-		}
 
-		if ( 0 == $sensei_course_loop['counter'] % $sensei_course_loop['columns'] ) {
-			$extra_classes[] = 'last';
+		// Apply "first" and "last" CSS classes for grid-based layouts.
+		if ( 1 !== $sensei_course_loop['columns'] ) {
+			if ( 0 == ( $sensei_course_loop['counter'] - 1 ) % $sensei_course_loop['columns'] || 1 == $sensei_course_loop['columns'] ) {
+				$extra_classes[] = 'first';
+			}
+
+			if ( 0 == $sensei_course_loop['counter'] % $sensei_course_loop['columns'] ) {
+				$extra_classes[] = 'last';
+			}
```

Comment:
Since we're editing these lines, we should probably switch to using `===`. Also, I think `1 == $sensei_course_loop['columns']` isn't necessary any more, right?

 ```suggestion
			if ( 0 === ( $sensei_course_loop['counter'] - 1 ) % $sensei_course_loop['columns'] ) {
				$extra_classes[] = 'first';
			}

			if ( 0 === $sensei_course_loop['counter'] % $sensei_course_loop['columns'] ) {
				$extra_classes[] = 'last';
			}
```

Reply:
(I'm not sure if Github's suggested change handled whitespace correctly... might need to be made outside of this)

Reply:
Fixed in https://github.com/Automattic/sensei/pull/3467/commits/f20683fda3d7eb6bfe6ffe7b06ec8d4f2065472b.